config/settings.yml

```diff
@@ -963,13 +963,13 @@
   :worker_monitor:
     :enforce_resource_constraints: false
     :kill_algorithm:
-      :name: :used_swap_percent_gt_value
+      :name: used_swap_percent_gt_value
```

Comment:
Does the caller not care if it's symbol or string? (same with the other algorithms).  Asking because I don't see a corresponding code change like what you did for poll_method.

Reply:
I just checked, it uses [send](https://github.com/ManageIQ/manageiq/blob/4cf45496b55360a8043510135a13427a57f64cd8/app/models/miq_server/worker_management/monitor/system_limits.rb#L119) so it should be ok.